src/components/passwordUpdate.js

```diff
@@ -51,13 +51,17 @@ const PasswordUpdate = () => {
               setMessage("An error occurred: " + response.statusText);
             }
           })
-          .catch((err) => setMessage(""));
+          .catch((err) => {
+            setMessage(err.response.data.error);
```

Comment:
In case response.data.error is not defined,  change to err.response?.data?.error  || err.response?.toString() || err.toString()

Reply:
Getting an eslint error with optional chaining notation and don't have time to dig into it right now I'm afraid